### PR TITLE
Asl 4299/add assessment uploads

### PR DESCRIPTION
**Pull request by joefitter**
### Uploads
* adds new [asl-attachments](https://github.com/UKHomeOffice/asl-attachments) upload path for harm benefit analysis files
* provides an accept/remove option
* shows up in project pages and task log

### 🏠 Housekeeping
* adds a `.prettierrc` so we can run prettier

### CODE REVIEW

File: pages/task/read/routers/confirm.js [hunk 1]
```diff
@@ -20,42 +24,71 @@ module.exports = () => {
...
-
-      if (values.status === 'intention-to-refuse') {
-        set(res.locals.static, 'inspector', req.user.profile);
-        set(res.locals.static, 'content.status.intention-to-refuse.action', 'Give reason for refusal');
-      }
-
-      next();
+  app.get('/', async (req, res, next) => {
+    const hbaTokenFromTask = req.task.data.meta.hbaToken;
+    console.log(req.task);
```
Comment (roc) [inline pages/task/read/routers/confirm.js:29]: remove console.log

File: pages/project/read/content/index.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 const { merge } = require('lodash');
 const baseContent = require('../../content');
 const downloadContent = require('../../../project-version/downloads/content');
+console.log(downloadContent);
```
Comment (roc) [inline pages/project/read/content/index.js:4]: remove console.log

File: pages/project/read/index.js [hunk 3]
```diff
@@ -48,91 +50,148 @@ module.exports = settings => {
...
       .catch(next);
   });
 
   app.get('/', (req, res, next) => {
-    res.enforcementModel = req.project;
+    if (!req.user.profile.asruUser) {
+      return next();
+    }
+
+    console.log(req.project.granted);
```
Comment (roc) [inline pages/project/read/index.js:113]: remove console.log

File: pages/task/read/routers/upload-hba.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+        if (!file) {
+          return next();
+        }
+        const formData = new FormData();
+
+        // File parsed by multer from incoming request
+        formData.append('file', file.buffer, file.originalname);
+
+        // TODO:
+        // * figure out how to correctly proxy attachment => localhost:8092
```
Comment (joefitter) [inline pages/task/read/routers/upload-hba.js:52]:
```suggestion
```

File: pages/task/read/routers/upload-hba.js [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+        const file = req.files?.upload?.[0];
+        if (!file) {
+          return next();
+        }
+        const formData = new FormData();
+
+        // File parsed by multer from incoming request
+        formData.append('file', file.buffer, file.originalname);
+
+        // TODO:
```
[the remark this answers is not in the document]
Comment (joefitter) [reply pages/task/read/routers/upload-hba.js:51]:
```suggestion
```

File: pages/task/read/routers/upload-hba.js [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+        // * figure out how to correctly proxy attachment => localhost:8092
+        // * once uploaded, attach to project/project version or task
+        // * replay attachment on inspection
```
Comment (joefitter) [inline pages/task/read/routers/upload-hba.js:54]:
```suggestion
```

File: pages/task/read/routers/upload-hba.js [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+          return next();
+        }
+        const formData = new FormData();
+
+        // File parsed by multer from incoming request
+        formData.append('file', file.buffer, file.originalname);
+
+        // TODO:
+        // * figure out how to correctly proxy attachment => localhost:8092
+        // * once uploaded, attach to project/project version or task
```
Comment (joefitter) [inline pages/task/read/routers/upload-hba.js:53]:
```suggestion
```

File: pages/project/read/content/index.js [hunk 8]
```diff
@@ -57,25 +58,30 @@ module.exports = merge({}, baseContent, {
...
     },
     revocation: {
       title: 'Revocation in progress',
-      description: 'A request to revoke this licence has been submitted and is being reviewed.'
+      description:
+        'A request to revoke this licence has been submitted and is being reviewed.'
     },
     'update-licence-holder': {
       title: 'Amendment in progress',
       // TODO: Devs do content
```
Comment (roc) [inline pages/project/read/content/index.js:75]:
```suggestion
```

Comment (roc) [inline pages/project/read/content/index.js]:
```suggestion
```

File: pages/project/read/content/index.js [hunk 9]
```diff
@@ -57,25 +58,30 @@ module.exports = merge({}, baseContent, {
...
-      description: 'A request to change the licence holder is currently in progress, you can view the task to see a list of available steps.'
+      description:
+        'A request to change the licence holder is currently in progress, you can view the task to see a list of available steps.'
     },
     'returned-draft': {
       title: 'Application in progress',
       // TODO: Devs do content
```
Comment (roc) [inline pages/project/read/content/index.js]:
```suggestion
```

**Pull request merged**